packages/wrangler/src/miniflare-cli/assets.ts

```diff
@@ -100,7 +100,7 @@ async function generateAssetsFetch(
 	let metadata = createMetadataObject({
 		redirects,
 		headers,
-		logger: log.warn,
+		logger: (message) => log.warn(message),
```

Comment:
If you do:

```suggestion
		logger = log.warn.bind(log)
```

Then I think you benefit from variadic argument support.

Reply:
Yup. This does just always take a single `message: string` arg, but you're probably right. That's a better, more future-proof, fix.